python/run-tests.py

```diff
@@ -40,7 +40,7 @@ def test(root_dir, package):
                    "--packages", package, test_file]
             print("Running tests in %s\n=============" % test_file)
             print("Command: %s" % str(cmd))
-            run_cmd(cmd, stream_output=True)
+            run_cmd(cmd, stream_output=True, print_cmd=False)
```

Comment:
why this change?

Reply:
why not print the cmd?

Reply:
we weren't doing it before. i added the print_cmd parameter to the run_cmd function.
since we weren't doing it before, saw no reason to do it now.

Reply:
might as well do it. better for debugging.

Reply:
sgtm